trulens_eval/tests/e2e/test_endpoints.py

```diff
@@ -206,6 +211,15 @@ def test_litellm_bedrock(self):
         # Litellm comes with cost tracking for bedrock though it may be inaccurate.
         self._test_llm_provider_endpoint(provider)
 
+    @optional_test
+    def test_cortex(self):
+        """Check that cost (token) tracking works for Cortex LLM Functions"""
+        from trulens_eval.feedback.provider.cortex import Cortex
```

Comment:
why do we even have the `__init__.py` file in `trulens_eval/feedback/provider` when it seems like wherever we're importing things from that directory we're just adding in the file like here anyway?

Reply:
We should not use aliases like the ones in `__init__.py` in trulens code but the aliases are there for trulens users' convenience.